### DISCUSSION TITLE
How do I properly uninstall it?

### DISCUSSION THREAD

**Discussion by helloyes3993** [noah-nuebling/mac-mouse-fix#612] [Questions]
Just wondering

**Comment by HurriedBiker** [comment]
Same need here. After runned it into the download folder, it's now bind to it and impossible to move the app in application folder. I tried to remove all binaries caches and preferences but a file keeps pointing on download folder.. Please advise.

Cheers

**Reply by ci7rix** [reply]
Have a look here : https://mousefix.org/about/
